README.md

```diff
@@ -187,7 +250,41 @@ Next, generate a Personal Access Token for bootstrapping the `core` packages, as
 
 Select your Gravatar icon on the top right corner of the Chef Habitat Builder on-prem web page, and then select **Profile**. This will take you to a page where you can generate your access token. Make sure to save it securely.
 
-## Bootstrap `core` packages
+## Bootstrap Builder with Habitat Packages (**New**)
+
+Chef Habitat Builder on-prem has no pre-installed package sets. You must populate your Builder instance by uploading packages.
+With Habitat *0.88.0*, two new commands were introduced to assist in bootstrapping an on-prem Builder instance with a set of stable packages:
+
+1. *hab pkg download*
+1. *hab pkg bulkupload*
+
+As you can see from the commands above, the package Bootstrap flow is comprised of two main phases: a download from the public [SaaS Builder](https://bldr.habitat.sh) followed by a bulkupload to your on-prem Builder instance(s). Historically, we bootstraped on-prem-builders by downloading all the packages in 'core' for all targets. That amounted to ~15GB and was both too much and too little, in that many of the packages weren't needed, and for many patterns (effortless) other origins were needed.
```

Comment:
Typo: "bootstraped"

Reply:
Fixed in https://github.com/habitat-sh/on-prem-builder/pull/212/commits/bf14ce0f10ff40c35f15c48f6122f570929a452d